pkg/services/temp_user/model.go

```diff
@@ -24,7 +24,7 @@ const (
 
 // TempUser holds data for org invites and unconfirmed sign ups
 type TempUser struct {
-	ID              int64 `xorm:"id"`
+	ID              int64
```

Comment:
in other tables we use these tags: `xorm:"pk autoincr 'id'"`
otherwise I think that xorm assumes for `i_d` column